### PR TITLE
Improve readme to enable people to use this code

### PR DESCRIPTION
**Pull request by ifireball**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,62 @@
...
+Installation
+------------
+```
+pip install redhat-opentelemetry-python-contrib
+```
+
+Using the file exporters
+------------------------
+The file exporters can be used by setting the `OTEL_*_EXPORTER` environment
+variables to `file`. Or using the equivalent arguments to
```
Comment (Omeramsc) [inline README.md]: replace dot after "`file`" to comma

File: README.md [hunk 2]
```diff
@@ -1,2 +1,62 @@
...
+`opentelemetry-instrument`.
+
+For example to export span data to file for a particular Python script:
```
Comment (Omeramsc) [inline README.md]: to **a** file*

**Review by Omeramsc — APPROVED** [review]
LGTM

**Comment by galitf** [comment]
LGTM

File: README.md [hunk 3]
```diff
@@ -1,2 +1,62 @@
...
+```
+opentelemetry-instrument --traces_exporter file python myscript.py
+```
+
+The file to which the data will be written to can be customized using the
+environment variables listed below. Otherwise,the listed default value would be
```
Comment (mafh314) [inline README.md]: Add whitespace after the `,`
Otherwise,  the listed ...

File: README.md [hunk 4]
```diff
@@ -1,2 +1,62 @@
...
+Using the script instrumentor
+-----------------------------
+Once installed, the script instrumentor will automatically wrap any Python
+script invoked with auto instrumentation enabled in a span that would include
+the script name, command-line arguments and exit status.
+
+The script instrumentor attempts to propagate the tracing context from the
+environment it was invoked in, by trying to read environment variables that are
+capitalized versions of the HTTP headers defined by the [W3C Trace Context
+specification][w3c]. This typically means that if the `TRACEPARNET` environment
```
Comment (mafh314) [inline README.md]: I think this should be `TRACEPARENT`

**Comment by mafh314** [comment]
LGTM

**Review by mafh314 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,2 +1,62 @@
...
+capitalized versions of the HTTP headers defined by the [W3C Trace Context
+specification][w3c]. This typically means that if the `TRACEPARENT` environment
+variable is defined in the environment the script runs in, the script span will
+become a child span of that trace. This is generally compatible with how other
+tools and systems handle things such as the [Ansible OpenTelemetry callback
+plugin][ans] and the [Jenkins OpenTelemetry plugin][jnk].
+
+The script instrumentor can cause traces to look a bit strange for things that
+are not meant to be stand-alone Python scripts such as Django and Flask server
+processes. It can be disabled bu setting the
```
Comment (Acepresso) [inline README.md:54]: Typo: disabled `bu`

**Pull request merged**